frontend/src/components/activity/CopyActivityInfoDialog.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <dialog-form
+    v-model="showDialog"
+    icon="mdi-content-copy"
+    :title="$tc('components.activity.copyActivityInfoDialog.title')"
+    :cancel-action="cancel"
+    :cancel-label="$tc('global.button.close')"
+  >
+    <template #activator="scope">
+      <slot name="activator" v-bind="scope" />
+    </template>
+
+    <p>
+      {{ $tc('components.activity.copyActivityInfoDialog.description') }}
+    </p>
+    <p v-if="clipboardReadState == 'prompt'">
+      <center>
+        <v-btn color="success" @click="requestClipboardAccess">
+          {{ $tc('components.activity.copyActivityInfoDialog.allow') }}
+        </v-btn>
+      </center>
+    </p>
+    <p v-if="clipboardReadState == 'granted'">
+      {{ $tc('components.activity.copyActivityInfoDialog.granted') }}
+    </p>
+    <p v-if="clipboardReadState == 'denied'">
+      {{ $tc('components.activity.copyActivityInfoDialog.denied') }}
+    </p>
+  </dialog-form>
+</template>
+
+<script>
+import DialogForm from '@/components/dialog/DialogForm.vue'
+import DialogBase from '@/components/dialog/DialogBase.vue'
+
+export default {
+  name: 'CopyActivityInfoDialog',
+  components: {
+    DialogForm,
+  },
+  extends: DialogBase,
+  data() {
+    return {
+      clipboardReadState: 'unknown',
```

Comment:
null is not good enough... or undefined?

Reply:
if displayed somewhere (debug); 'unknown' more understandable